src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
-// write code here
+const tree = document.querySelector('.tree');
+
+[...document.querySelectorAll('li')].map((li) => {
```

Comment:
You can avoid creating a new array using the `map` method. This method returns a new array but does not mutate the existing one.

But here you don't need to use `map`. This method only returns a new array based on the existing one. Here you have to use `forEach` method instead of `map`.